.github/workflows/node.js.yml

```diff
@@ -15,7 +15,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 16.x]
+        node-version: [14.x, 16.x, 18.x]
```

Comment:
node 14 will actually reach EOL next month (April 2023). It might be worth demoting it now

Reply:
Good point